src/variableDiagnosticsProvider.ts

```diff
@@ -51,23 +45,23 @@ export class VariableDiagnosticsProvider {
 
         let existArray = await VariableProcessor.checkVariableDefinitionExists(document, varNames);
 
-        existArray.forEach((ea) => {
-            if (!ea.exists) {
+        existArray.forEach(({name, exists}) => {
+            if (!exists) {
                 vars.forEach((v) => {
-                    if (v.variableName === ea.name) {
+                    if (v.variableName === name) {
                         diagnostics.push({
                             severity: DiagnosticSeverity.Error,
                             range: new Range(new Position(v.lineNumber, v.startIndex), new Position(v.lineNumber, v.endIndex)),
                             message: `${v.variableName} is not loaded in memory`,
-                            source: 'ex',
-                            code: "10",
+                            source: 'http',
+                            code: 0,
```

Comment:
do we need this line, what is `code` used for?

Reply:
The `code` parameter is required, so I have to set it to something